content/public/common/BUILD.gn

```diff
@@ -259,6 +259,9 @@ jumbo_source_set("common_sources") {
     "//ui/surface",
     "//url/ipc:url_ipc",
   ]
+  if (enable_castanets) {
+    public_deps += [ "//third_party/wrt/ewk/efl_integration:chromium-ewk" ]
```

Comment:
This static library linked with each modules, not once?
Please check the wrt libraries are linked only once in renderer target.

Reply:
This statement has been removed. We no longer uses chromium-ewk related symbol in content/public/common.